docs/excel2xml-module.md

```diff
@@ -258,6 +258,36 @@ make_text_prop(
 ```
 
 
+#### The Special Properties of `<video-segment>` and `<audio-segment>`
+
+`<video-segment>` and `<audio-segment>` work differently than normal resources.
```

Comment:
I would make a note in regards to what needs to happen or if nothing needs to happen in the json.

Reply:
Done with https://github.com/dasch-swiss/dsp-tools/commit/b44e5c2b17d774ffcbc8a1f60d94ae3a97eaea83